src/chapters/orm-relationships/many-to-many.rst

```diff
@@ -0,0 +1,332 @@
+
+.. sourcecode:: java
+   :lineno-start: 11
+
+   public class EventTagDTO {
+
+      @NotNull
+      private Event event;
+
+      @NotNull
+      private Tag tag;
+
+      public EventTagDTO() {}
+
+      public Event getEvent() {
+         return event;
+      }
+
+      public void setEvent(Event event) {
+         this.event = event;
+      }
+
+      public Tag getTag() {
+         return tag;
+      }
+
+      public void setTag(Tag tag) {
+         this.tag = tag;
+      }
+   }
+
+We place this class in a new package, ``dto``, contained within the ``models`` package. It is a model since it structures data that our application uses. However, it is not persistent (there is no ``@Entity`` annotation) because we won't need to store it in the database. 
+
+Connecting Two Objects 
+^^^^^^^^^^^^^^^^^^^^^^
+
+The process of connecting two many-to-many objects is similar to that of connecting objects with other types of relationships. We need a form that can allow a user to create a new relationship, and processing the form should result in the relationship being saved to the database.
+
+Rendering the Form
+++++++++++++++++++
+
+Given a specific event, we want to be able to render a form that allows the user to add a tag to that event. This means that the form has to know *which event* the user wants to work with. One way to do this is with query parameters. 
+
+For example, say a user wants to add a tag to the ``Event`` object with ID 13. To get to the form that enables this, they may navigate to ``/events/add-tag?eventId=13``. This request will return a form that allows a tag to be added to *only* the event with ID 13.
+
+Here is the handler method in ``EventController`` that renders the form, broken down line-by-line just below.
+
+.. sourcecode:: java
+   :lineno-start: 112
+
+   @GetMapping("add-tag")
+   public String displayAddTagForm(@RequestParam Integer eventId, Model model){
+      Optional<Event> result = eventRepository.findById(eventId);
+      Event event = result.get();
+      model.addAttribute("title", "Add Tag to: " + event.getName());
+      model.addAttribute("tags", tagRepository.findAll());
+      EventTagDTO eventTag = new EventTagDTO();
+      eventTag.setEvent(event);
+      model.addAttribute("eventTag", eventTag);
+      return "events/add-tag.html";
+   }
+
+- **Line 112**: Specifies that the handler will be available at the route ``/events/add-tag``, and will respond to ``GET`` requests.
+- **Line 113**: Defines the ``displayAddTagForm`` handler, which has a required query parameter, ``eventId``.
+- **Line 114**: Queries the repository for the ``Event`` object with ID equal to the value of ``eventId``.
+- **Line 115**: Extracts the ``Event`` object from the result of the query. We would ideally include a conditional to check that such an object exists before proceeding.
```

Comment:
We can include some language explaining why this is not included here (for space, for focus on the DTO usage?)